web/handlers/admin/members.go

```diff
@@ -104,7 +104,8 @@ func (h membersHandler) changeRole(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	http.Redirect(w, req, redirectToMembers, http.StatusTemporaryRedirect)
+	memberDetailsURL := fmt.Sprint("/admin/member?id=", memberID)
```

Comment:
```suggestion
	urlTo := web.NewURLTo(router.CompleteApp())
	memberDetailsURL := urlTo(router.AdminMemberDetails, "id", memberID).String()
```

this way we are safe when we change the layout of the paths.

Reply:
might need to add imports for package `.../web` and `.../web/router` but can't suggest changes outside of the diff... :upside_down_face:

Reply:
Thanks for the suggestion. I did it in a separate commit, and added the imports